code/modules/mob/living/living.dm

```diff
@@ -492,24 +492,29 @@
 				var/obj/effect/proc_holder/spell/spell = S
 				spell.updateButtonIcon()
 
+/mob/living/proc/remove_CC(should_update_mobility = TRUE)
+	SetStun(0, FALSE)
+	SetKnockdown(0, FALSE)
+	SetImmobilized(0, FALSE)
+	SetParalyzed(0, FALSE)
+	SetSleeping(0, FALSE)
+	setStaminaLoss(0)
+	SetUnconscious(0, FALSE)
+	bodytemperature = BODYTEMP_NORMAL
```

Comment:
No, don't fuck with body temp, find a different way to suppress that slowdown

Reply:
https://github.com/tgstation/tgstation/blob/c0c324a4693837d474cfcd0bf2b293bf3023e403/code/modules/mob/living/carbon/human/species.dm#L1121

although I figured this was to avoid both the slowdown AND the damage associated with unsafe body temp.

Reply:
It's a tricky one, ideally it should just temporarily prevent slowdowns from low temperature and give the user a chance to warm up

Reply:
Since this proc is supposed to remove CC and CC alone

Reply:
alternatively pull it out of the proc and put it on the item? The slowdown check should still be added here to counter the slowdown but I think there is some use in a body temp reset for legion cores.

Reply:
Personally I'd prefer pulling it out of the proc - giving temporary immunity to the slowdown would allow you to effectively tank watcher shots after healing one, given that the damage isn't the main threat from watchers.